### ISSUE TITLE
Installation procedure

### ISSUE THREAD

**Issue by jacobsamlarose** [ruundii/bthidhub#48]
Noob here. Little confused about the installation instructions. Readme says to copy wpa_supplicant.conf and ssh file to the root of the boot partition and then (following the Mac instructions) run setup.sh. Am I supposed to place setup.sh in the root of the boot partition, too?

**Comment by Wheels00** [comment]
No, if I remember correctly, once you boot the pi you run ssh pi@[rpi-ip-address] 'bash -s' < setup.sh

Which ssh's into the pi and runs the setup.sh file from your mac which then installs bthidhub to the pi over ssh

**Comment by Dreamsorcerer** [comment]
These days you can just use the Pi Imager to install, which is less likely to cause you confusion.
